playground/samples/single.js

```diff
@@ -5,4 +5,5 @@ module.exports = {
   },
   formData: "initial value",
   uiSchema: {},
+  extraErrors: [],
```

Comment:
Why do you need an empty extraErrors prop for each item in the playground?

Reply:
That way it overwrites it every time you switch